satpy/readers/yaml_reader.py

```diff
@@ -426,8 +426,9 @@ def check_file_covers_area(file_handler, check_area):
         from satpy.resample import get_area_def
         try:
             gbb = Boundary(*file_handler.get_bounding_box())
```

Comment:
Are there any requirements for the order of the boundary elements or the number of elements?

Reply:
the order should be clockwise around the area, no real limit on the number of elements